package.json

```diff
@@ -25,22 +25,21 @@
     "express": "^4.16.3",
     "express-prom-bundle": "^5.1.5",
     "google-protobuf": "3.5.0",
-    "lodash": "^4.17.19",
+    "lodash": "^4.17.21",
     "minimist": "^1.2.0",
     "morgan": "^1.9.0",
-    "mz": "^2.7.0",
     "prom-client": "^11.5.3",
-    "puppeteer": "^2.0.0",
-    "puppeteer-cluster": "^0.18.0",
+    "puppeteer": "^3.3.0",
+    "puppeteer-cluster": "^0.22.0",
     "unique-filename": "^1.1.0",
     "winston": "^3.2.1"
   },
   "devDependencies": {
     "@types/express": "^4.11.1",
-    "@types/node": "^10.0.9",
-    "husky": "^3.1.0",
+    "@types/node": "^14.14.41",
+    "husky": "^4.3.8",
     "lint-staged": "^9.5.0",
-    "pkg": "4.4.8",
+    "pkg": "^5.1.0",
```

Comment:
Had quite some problems upgrading this one earlier. But I'll test out the built linux binaries as a start to verify. We should probably ask someone running Mac to test it out. The binaries can be downloaded from the CircleCI build, for example here for linux: https://app.circleci.com/pipelines/github/grafana/grafana-image-renderer/268/workflows/7c9e54c4-a452-440e-9475-cbb65d37487e/jobs/833/artifacts

Reply:
Tested by rendering a few reports and worked without issues.